guides/07_streaming/02_object-detection-from-webcam.md

```diff
@@ -0,0 +1,93 @@
+# Object Detection from a Webcam Stream
```

Comment:
This is basically the only change in this PR apart from the demo. 

I think this is a good starting point. As discussed earlier today, I think we can improve some things internally to improve the experience on spaces.